packages/cli-link-assets/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
```

Comment:
nit: you can link CLI's [CONTRIBUTING.md](https://github.com/react-native-community/cli/blob/main/CONTRIBUTING.md) guide

Reply:
Could you elaborate more @szymonrybczak ? I followed the way we've done for other files, e.g [here](https://github.com/react-native-community/cli/blob/main/packages/cli-clean/CHANGELOG.md) and [here](https://github.com/react-native-community/cli/blob/main/packages/cli-config/CHANGELOG.md).

Reply:
Ah, I didn't know that we have CHANGELOGs in this repository 😅 Let's leave it as it is then 👍